samcli/lib/providers/cfn_api_provider.py

```diff
@@ -85,17 +85,17 @@ def _extract_cloud_formation_route(self, logical_id, api_resource, collector, cw
     @staticmethod
     def _extract_cloud_formation_stage(resources, stage_resource, collector):
         """
-        Extract the stage from AWS::ApiGateway::Stage resource by reading and adds it to the collector.
-        Parameters
-       ----------
-        resources: dict
-            All Resource definition, including its properties
+         Extract the stage from AWS::ApiGateway::Stage resource by reading and adds it to the collector.
```

Comment:
is there some formatting issues here?

Reply:
It was a forced reformatting by black 20, but since I switched back to black 19 I don't think either one will cause pr problems. but i guess i'll just leave it this way